docs/src/main/paradox/scala/http/routing-dsl/rejections.md

```diff
@@ -31,19 +31,14 @@ the @ref[handleRejections](directives/execution-directives/handleRejections.md)
 <a id="the-rejectionhandler"></a>
 ## The RejectionHandler
 
-The @ref[handleRejections](directives/execution-directives/handleRejections.md) directive delegates the actual job of converting a list of rejections to its argument, a
-@scaladoc[RejectionHandler](akka.http.scaladsl.server.RejectionHandler), which is defined like this:
-
-```scala
-trait RejectionHandler extends (immutable.Seq[Rejection] => Option[Route])
-```
-
-Since a `RejectionHandler` returns an `Option[Route]` it can choose whether it would like to handle the current set
-of rejections or not. If it returns `None` the rejections will simply continue to flow through the route structure.
+The @ref[handleRejections](directives/execution-directives/handleRejections.md) directive delegates the actual job of converting a list of rejections to the provided @scala[@scaladoc[RejectionHandler](akka.http.scaladsl.server.RejectionHandler)]@java[@javadoc[RejectionHandler](akka.http.javadsl.server.RejectionHandler)],
+so it can choose whether it would like to handle the current set of rejections or not.
+Unhandled rejections will simply continue to flow through the route structure.
```

Comment:
I rewrote this paragraph to be a bit higher-level and similar to the Java version since there's a link to the API doc if type info is needed and the section on custom rejection handlers go into depth with code examples.